lib/index.js

```diff
@@ -45,13 +45,16 @@ function generate_google_oauth2_url () {
 
 exports.register = function googleauth (server, options, next) {
   OPTIONS = options;
+  OPTIONS.config = OPTIONS.config || {};
+  OPTIONS.config.auth = false;
```

Comment:
Hmm, I thought that it would be more stable to save the auth option as it was before but, actually, this is a very good point. Thus, having an options object without mentioning in the documentation that auth is always set to false is a little bit smelly to me.

A real-world scenario (with enabled auth) would be if I want to merge accounts from different strategies, for example.